bios/brianatssat.md

```diff
@@ -0,0 +1,4 @@
+## brianatssat
+- IT Director at SSAT
+- Wants to learn more about Git
```

Comment:
Are you signed up for tomorrow's Intermediate class? Today we spend more time in GitHub.com and the desktop apps, but tomorrow we will spend most of our time on the command line with git.